toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/TraceResultScreen.kt

```diff
@@ -75,7 +86,12 @@ internal fun TraceResultScreen(
             .fillMaxSize()
             .padding(horizontal = 10.dp)) {
 
+            if (traceResults.isEmpty()) {
+                return@Column
+            }
+
             val selectedTraceRun = traceResults[selectedTraceRunIndex]
```

Comment:
remembering `selectedTraceRun` fixes the issue: 
```suggestion
            val selectedTraceRun = remember(selectedTraceRunIndex) {  traceResults[selectedTraceRunIndex]}
```